flink-connector-jdbc/pom.xml

```diff
@@ -42,6 +42,10 @@ under the License.
 		<oracle.version>21.8.0.0</oracle.version>
 		<trino.version>418</trino.version>
 		<byte-buddy.version>1.12.10</byte-buddy.version>
+		<surefire.module.config> <!-- required by
+		Db2ExactlyOnceSinkE2eTest --> --add-opens=java.base/java.util=ALL-UNNAMED <!--
+		SimpleJdbcConnectionProviderDriverClassConcurrentLoadingITCase--> --add-opens=java.base/java.lang=ALL-UNNAMED
+		</surefire.module.config>
```

Comment:
Why is is this a part of the PR?
It is already in main

I guess you need to rebase